src/components/Table/GridTable.stories.tsx

```diff
@@ -634,3 +635,41 @@ export function WrappedHeaders() {
     />
   );
 }
+
+type DataRow = { kind: "data"; name: string; role: string; date: string; priceInCents: number };
+type TotalsRow = { kind: "total"; totalPriceInCents: number };
+type ColspanRow = HeaderRow | DataRow | TotalsRow;
+export function ColSpan() {
+  const nameCol = column<ColspanRow>({
+    header: "Name",
+    data: ({ name }) => name,
+    total: () => ({ content: "Totals:", colspan: 3, alignment: "right" }),
+  });
+  const detailCol = column<ColspanRow>({ header: "Details", data: ({ role }) => role, total: "" });
+  const dateCol = dateColumn<ColspanRow>({ header: "Date", data: ({ date }) => date, total: "" });
```

Comment:
Kind of a bummer to always need to put a `total` property in here as we're just going to span over it. But not sure how else this could know?

Reply:
...right, because it depends on the order that the columns are rendered in.

Wonder if we could have like an `export const skippedColumn = ...` sort of thing that by convention we use in these spots to indicate "yes we know it's being skipped"?

Or maybe something like:

```
const detailCol = column<ColspanRow>({
  header: "Details",
  data: () =>,
  ...ignoreColspan("total", "otherKind"),
});
```

And `ignoreColspan` would just create a hash with `total: () => throw new Error("should not render")` or something like that...

Reply:
Pretty good to leave this as-is for now and figure out something cuter later.